test/fail/lambdas-ok.mo

```diff
@@ -58,6 +58,25 @@ func foo<T, A>(t : T, _ : A, _ : A -> T) : T = t;
 // In the 2nd round: Int <: T  (solve T := Int)
 let _ = foo(1, "abc", func _ = -1);
 
+module ClosedBody1 {
+  func f1<A>(_ : A, _ : A -> Int) {};
+  func f2<A, B>(_ : A, _ : A -> Int) {};
+  // It should defer the func, solve A=Nat in the 1st round, but (1 : Int) should be checked and leave nothing to solve in the 2nd round
+  func _main() {
+    let _ = f1(1, func _ = 1);
+    let _ = f2(1, func _ = 1); // extra unused type variable B
```

Comment:
These both work because in the first case it doesn't matter what we chose for B and the second B is determined by expected return type, right?

Reply:
Ok -  I no longer understand my original comment ;->